milestone2.ts

```diff
@@ -157,7 +161,7 @@ console.log(kyleNew);
 
 type GetVolunteerInfo = (vol: Volunteer) => String;
 let getVolunteerInfo: GetVolunteerInfo = (vol) => {
-  let [name, age, city] = [vol.name, vol.age, vol.city]; // Code here
+  let { name, age, email, position, city, state } = vol; // Code here
```

Comment:
Good job using object destructuring here, but for future reference we only want to grab out the fields that we will be using, you aren't required to destructure the whole object.